test/bugsnag/payload_test.exs

```diff
@@ -6,7 +6,7 @@ defmodule Bugsnag.PayloadTest do
     try do
       # If the following line is not on line 9 then tests will start failing.
       # You've been warned!
-      Harbour.cats(3)
+      raise "an error occurred"
```

Comment:
NOTE: I made this a RuntimeError since the original error (UndefinedFunctionError) is more like a special case. So I made this a more general case.